routes/web.php

```diff
@@ -28,6 +28,8 @@
 Route::get('/auth/logout', [AuthController::class, 'logout'])->name('auth.logout');
 Route::get('/auth/edit', [AuthController::class, 'edit'])->name('auth.edit');
 Route::post('/auth/update', [AuthController::class, 'update'])->name('auth.update');
+Route::get('/auth/resetPassword', [AuthController::class, 'viewReset'])->name('auth.viewChangePassword');
```

Comment:
@prince06031998 
line 31 endpoint is "/auth/resetPassword"
but line 32 endpoint is "/auth/resetpassword" 

Please make those be consistent